sdk/include/aditof/camera.h

```diff
@@ -279,6 +279,20 @@ class SDK_API Camera {
      * @return Status
      */
     virtual Status adsd3500GetRadialThresholdMax(int &threshold) = 0;
+
+    /**
+     * @brief Get the sensor temperature threshold
```

Comment:
I think the work "threshold" is not necessary.

Reply:
removed